gaas-ui/src/components/schema-builder/add-type.tsx

```diff
@@ -5,6 +5,7 @@ import AddAggregateFunction from "./add-aggregate-function";
 import ClearIcon from "@material-ui/icons/Clear";
 import AddSerialiser from "./add-serialiser";
 import AddValidateFunctions from "./add-validate-functions";
+import sanitizeInputs, { isJSONString } from "../../util/sanitize-inputs";
 
 interface IProps {
```

Comment:
So we don't need to send a Description when creating a new graph?

Reply:
You can create a graph with no description, and you can also create types, entities and edges without descriptions. It is nice to have the option, but they don't need to be mandatory. Perhaps an issue for another ticket.

Reply:
I've created a ticket #272 for this

Reply:
Hey @t92549 we're going to keep description mandatory in GaaS as it will help support the GaaS platform when its deployed to production. This way we'd know why a graph has been created and what for purpose.